routers/api/v1/repo/migrate.go

```diff
@@ -137,12 +118,29 @@ func Migrate(ctx *context.APIContext) {
 		return
 	}
 
+	form.LFS = form.LFS && setting.LFS.StartServer
+
+	if form.LFS && len(form.LFSEndpoint) > 0 {
+		ep := lfs.DetermineEndpoint("", form.LFSEndpoint)
+		if ep == nil {
+			ctx.Error(http.StatusInternalServerError, "", ctx.Tr("repo.migrate.invalid_lfs_endpoint"))
+			return
+		}
+		err = migrations.IsMigrateURLAllowed(form.LFSEndpoint, ctx.User)
```

Comment:
Is it possible that we need to run this on the ep rather than the form.LFSEndpoint?

Reply:
Good point. `DetermineEndpoint` doesn't change much but I will change it.